pkg/processor/sourceprocessor/README.md

```diff
@@ -84,3 +84,35 @@ prefixed with the value defined in `keys.annotation_prefix` config option.
 This can be achieved with the [Kubernetes processor](../k8sprocessor).
 
 For example, if a resource has the `k8s.pod.annotation.sumologic.com/exclude` attribute, the resource will be dropped.
+
+### Container-level pod annotations
+
+To make it possible to set different metadata on logs from different containers inside a pod,
```

Comment:
Do we mention it somewhere in the README what's the precedence between pod and container-level annotations?

Reply:
Added note https://github.com/SumoLogic/sumologic-otel-collector/pull/260/commits/3acb6e8688cff8f956f2bb9a33d39b827f9e53a9